src/kmer.rs

```diff
@@ -4,10 +4,7 @@
 /// Returns true if the base is a unambiguous nucleic acid base (e.g. ACGT) and
 /// false otherwise.
 fn is_good_base(chr: u8) -> bool {
-    match chr as char {
-        'a' | 'c' | 'g' | 't' | 'A' | 'C' | 'G' | 'T' => true,
-        _ => false,
-    }
+    matches!(chr as char, 'a' | 'c' | 'g' | 't' | 'A' | 'C' | 'G' | 'T')
```

Comment:
is that clippy asking for that change?

Reply:
Yes. My first commit (6f6b482) failed CI because of this warning. It's neat. I didn't even realise that macro existed. One of the few circumstances I've seen where a macro actually *adds* readability.